### PR TITLE
Added ach-UG translation to the requirements

### PR DESCRIPTION
**Pull request by bdunogier**
> [EZP-26829](https://jira.ez.no/browse/EZP-26829)
> Related to https://github.com/ezsystems/PlatformUIBundle/pull/773 and https://github.com/ezsystems/ezpublish-kernel/pull/1889

Required by in-context translation.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
         "ezsystems/ezplatform-solr-search-engine": "^1.1@dev",
         "ezsystems/platform-ui-bundle": "~1.5@dev",
         "ezsystems/ez-support-tools": "~0.1.0",
+        "ezplatform-i18n/ezplatform-i18n-ach_ug": "^1.0@dev"
         "egulias/listeners-debug-command-bundle": "~1.9",
```
Comment (andrerom) [inline composer.json]: getting emails from packagist about this thing :)

Comment (bdunogier) [inline composer.json]: hmmm what ? :)

Comment (bdunogier) [inline composer.json]: Fixed, thank you :)

**Comment by bdunogier** [comment]
Added a requirement for bazinja JS translation bundle 2.7@dev, because we need https://github.com/willdurand/BazingaJsTranslationBundle/pull/191. I have asked for a 2.6 release that would include this change as well.

**Pull request merged**